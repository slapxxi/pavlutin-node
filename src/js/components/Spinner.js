import React from 'react';
import Spin from './Spinner.styled';

function Spinner({ color }) {
  return <Spin color={color}>Loading...</Spin>;
}

export default Spinner;
