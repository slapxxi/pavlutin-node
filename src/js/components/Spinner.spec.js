import React from 'react';
import render from 'react-test-renderer';
import Spinner from './Spinner';

it('renders spinner', () => {
  const tree = render.create(<Spinner />);
  expect(tree.toJSON()).toMatchSnapshot();
});

it('renders spinner with color', () => {
  const tree = render.create(<Spinner color="blue" />);
  expect(tree.toJSON()).toMatchSnapshot();
});
