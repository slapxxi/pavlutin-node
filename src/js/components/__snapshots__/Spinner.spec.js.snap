// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders spinner 1`] = `
<span
  className="jtFWAu"
  color={undefined}
>
  Loading...
</span>
`;

exports[`renders spinner with color 1`] = `
<span
  className="dHrOlN"
  color="blue"
>
  Loading...
</span>
`;
